src/main/java/org/reaktivity/nukleus/http/internal/stream/ServerAcceptStream.java

```diff
@@ -421,6 +429,21 @@ private void processEnd(
         doEnd();
     }
 
+    private void processAbort()
+    {
+        factory.correlations.remove(acceptCorrelationId);
+        if (targetBeginIssued)
+        {
+            factory.writer.doAbort(target, targetId);
+            connectionReplyCleanup.run();
+        }
+        else
+        {
+            correlation.state().doAbort(factory.writer);
```

Comment:
Why is `doAbort` on the `correlation` state?

Reply:
Fixed it.